e2e/po/app.filters.ts

```diff
@@ -0,0 +1,66 @@
+import { browser, by, element, ExpectedConditions, promise } from 'protractor';
+
+export class Filters {
+  DETERMINISTIC_WAIT = 3000; // in milliseconds
+  retryAttempts = 0;
```

Comment:
why this is 0? this should be set to MAX here and we should try in test until exhausted. e.g. retryAttempts-- and retryAttempts > 0